nexus-sdk-js/src/File/__tests__/file.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { resetMocks, mock, mockResponse } from 'jest-fetch-mock';
+import Nexus, { File as NexusFile } from '../..';
+import { FileResponse } from '../types';
+
+const baseUrl = 'http://api.url';
+Nexus.setEnvironment(baseUrl);
+
+const mockPostFileResponse: FileResponse = {
+  '@context': 'https://bluebrain.github.io/nexus/contexts/resource.json',
+  '@id': 'base:d8848d4c-68f7-4ffd-952f-63a8cbcb86a9',
+  '@type': 'File',
+  _bytes: 670,
+  _digest: {
+    _algorithm: 'SHA-256',
+    _value: '25fc54fba0beec17a598b5a68420ded1595b2f76f0a0b7c6077792ece828bc2e',
+  },
+  _filename: 'myfile.jpg',
+  _mediaType: 'image/jpeg',
+  _self:
+    'https://nexus.example.com/v1/files/myorg/myproj/base:d8848d4c-68f7-4ffd-952f-63a8cbcb86a9',
+  _constrainedBy: 'https://bluebrain.github.io/nexus/schemas/file.json',
+  _project: 'https://nexus.example.com/v1/projects/myorg/myproj',
+  _rev: 1,
+  _deprecated: false,
+  _createdAt: '2019-01-28T12:15:33.238Z',
+  _createdBy: 'https://nexus.example.com/v1/anonymous',
+  _updatedAt: '2019-01-28T12:15:33.238Z',
+  _updatedBy: 'https://nexus.example.com/v1/anonymous',
+};
+
+function testClassProperties(file: NexusFile, response: FileResponse) {
```

Comment:
You can use a snapshot for this, that's what they are for. Generate the snapshot and make sure it looks ok.